src/main/java/frc/robot/commandgroups/JustShoot.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot.commandgroups;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.conveyor.Conveyor;
+import frc.robot.subsystems.conveyor.commands.Convey;
+import frc.robot.subsystems.flap.Flap;
+import frc.robot.subsystems.hood.Hood;
+
+public class JustShoot extends SequentialCommandGroup {
+    public JustShoot(Conveyor conveyor, Flap flap, Hood hood) {
+        addCommands(new Convey(conveyor, -0.25).withTimeout(0.075));
+        addCommands(new InstantCommand(() -> RobotContainer.shooting = true));
+//        if (RobotContainer.hardCodedVelocity)
```

Comment:
Why are these comments necessary?

Reply:
These comments replace the hard coded and safeties from before. It might be good to uncomment, but first we need to see the regular version works

Reply:
I think we should just uncomment it shouldn't change the logic of the code if these safeties are turned off